site/src/components/mdx/faq.tsx

```diff
@@ -75,7 +75,14 @@ export const FAQ: FunctionComponent<FAQProps> = ({
           <Typography ref={headingRef}>{question}</Typography>
         </AccordionSummary>
       </Link>
-      <AccordionDetails sx={{ padding: 0, marginTop: 1, marginBottom: 4 }}>
+      <AccordionDetails
+        sx={{
+          padding: 0,
+          marginTop: 1,
+          marginBottom: 4,
+          "& > *": { visibility: "inherit" },
```

Comment:
Accordion content wrapper is hidden when the accordion was closed but its content still has `visibility: visible` even though the content isn't visible on screen. This ensures the content inherits the visibility value of its parent